app/views/auctions/_list_item.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="usa-grid">
     <div class='usa-width-one-whole issue-label'>
       <h3 class='issue-title'>
-        <span class="usa-label-big <%= auction_label_class(auction) %>"><%= auction_label(auction) %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
+        <span class="usa-label-big <%= auction.label_class %>"><%= auction.label %></span> <a href="<%= auction_path(auction) %>"><%= auction.title %></a>
```

Comment:
Yeah I was wondering how these things were going to divide up too.

I put the stuff in the view models that were specific to the individual action/view. In this case we are in a list item for an auction. It is really in the realm of a single object that is potentially used in both the show and the index views. I was feeling at first that these methods seemed so very view related I didn't want to throw them into the presenter, and maybe there is a delegate class that could be extracted from the presenter and put into the view model. I just didn't see a sensible way to do that in the end.